src/components/common/errors/404.njk

```diff
@@ -1,6 +1,12 @@
 {% extends "common/layout/base.njk" %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% set activeNav = '404' %}
 {% set pageTitleName = 'error.error404.title' | translate %}
+{% if isUserLoggedIn %}
+  {% set buttonText = button_text | default('error.error404.content.signInUserButtonText' | translate, true) %}
```

Comment:
You are right, I'm baffled as well because button_text it is on 13 templates currently. Do you thing we need a separate ticket to investigate those rather than try and fix it in this one?

Reply:
Yeah you're right, makes sense to address it in a separate ticket since there are so many instances of it! I'll resolve this thread and document it elsewhere.